apps/beeai-web/src/acp/client.ts

```diff
@@ -14,16 +14,16 @@
  * limitations under the License.
  */
 
-import "server-only";
-import { Client } from "@i-am-bee/acp-sdk/client/index";
-import type { ServerCapabilities } from "@i-am-bee/acp-sdk/types";
-import { SSEClientTransport } from "@i-am-bee/acp-sdk/client/sse";
-import { ACP_CLIENT_SERVER_URL } from "@/constants";
-import { fetch as undiciFetch, BodyInit } from "undici";
+import 'server-only';
+import { Client } from '@i-am-bee/acp-sdk/client/index';
+import type { ServerCapabilities } from '@i-am-bee/acp-sdk/types';
+import { SSEClientTransport } from '@i-am-bee/acp-sdk/client/sse';
+import { ACP_CLIENT_SERVER_URL } from '@/constants';
+import { fetch as undiciFetch, BodyInit } from 'undici';
```

Comment:
How about using:
a) https://github.com/trivago/prettier-plugin-sort-imports or
b) https://www.npmjs.com/package/@ianvs/prettier-plugin-sort-imports

To take care of imports order as well?